pages/[lng]/gift-card/index.tsx

```diff
@@ -1,7 +1,7 @@
 /* library package */
 import { FC } from 'react'
 import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
-import { GiftCard, useI18n } from '@sirclo/nexus'
+import { GiftCard, useI18n, useAuthToken } from '@sirclo/nexus'
```

Comment:
```suggestion
import { 
  GiftCard,
  useI18n,
  useAuthToken
} from '@sirclo/nexus'
```

legenda mengatakan jika improtnya ada 3 atau lebih dibikin ke bawah seperti contoh

berlaku buat yang di file lain jika ada